### ISSUE TITLE
Error trying to setup multitool from MODULE.bazel

### ISSUE THREAD

**Issue by mvgijssel** [theoremlp/rules_multitool#8]
In https://github.com/vgijssel/setup/pull/609/commits/0d983b866658fb9e3190a9f5b552d3a6eb840bda I'm trying to use rules_multitool to wrap the github_cli. I'm using a module extension to load in the necessary archives and loading those archives using the multitool. Unfortunately I'm running into the following error:

```shell
(devbox) nix-shell-env ❯ bazel run @rules_release//:publish_github_release
ERROR: Traceback (most recent call last):
        File "/private/var/tmp/_bazel_maarten/8c4bfcd606c4d7b95cf418885251b08c/external/rules_release~override/extensions.bzl", line 4, column 17, in _non_module_dependencies_impl
                dependencies()
        File "/private/var/tmp/_bazel_maarten/8c4bfcd606c4d7b95cf418885251b08c/external/rules_release~override/repositories.bzl", line 34, column 14, in dependencies
                multitool(
        File "/private/var/tmp/_bazel_maarten/8c4bfcd606c4d7b95cf418885251b08c/external/rules_multitool~0.1.1/multitool/private/multitool.bzl", line 61, column 31, in multitool
                native.register_toolchains(*[
Error in register_toolchains: The native module can be accessed only from a BUILD thread. Wrap the function in a macro and call it from a BUILD file
ERROR: Analysis of target '@rules_release~override//:publish_github_release' failed; build aborted: error evaluating module extension non_module_dependencies in @rules_release~override//:extensions.bzl
INFO: Elapsed time: 2.530s
INFO: 0 processes.
ERROR: Build failed. Not running target
FAILED: Build did NOT complete successfully (77 packages loaded, 3023 targets configured)
    Fetching repository @bazel_tools~xcode_configure_extension~local_config_xcode; Fetching the default Xcode version
```

Can't I use the multitool from a module extension?

**Comment by mark-thm** [comment]
I don't think you'll be able to use multitool from a module extension at the moment; I'm a bit stuck on how to translate the register_toolchains calls to more appropriate use_repo calls, unfortunately.

**Comment by mark-thm** [comment]
Fixed by #9
